include/segment_symbols.h

```diff
@@ -84,6 +84,12 @@ DECLARE_SEGMENT(code)
 DECLARE_ROM_SEGMENT(code)
 DECLARE_BSS_SEGMENT(code)
 
+// N64-only, not wrapped in these are not wrapped in an `#if PLATFORM_N64`
+// so that the N64DD code can always be built.
+DECLARE_SEGMENT(n64dd)
+DECLARE_ROM_SEGMENT(n64dd)
+DECLARE_BSS_SEGMENT(n64dd)
```

Comment:
I disagree with putting everything into segment_symbols, this header is just as good as variables.h or functions.h

but whatever for now